supplemental_content/views.py

```diff
@@ -42,3 +45,38 @@ def get_queryset(self):
                 )
             ).distinct()
         return query
+
+
+class SupplementalContentSectionsView(generics.CreateAPIView):
+    def create(self, request, *args, **kwargs):
+        for section in request.data["sections"]:
+            try:
+                new_section, created = Section.objects.get_or_create(
+                            title=section["title"],
+                            part=section["part"],
+                            section_id=section["section"]
+                        )
+            except Exception:
+                return Response({'error': True, 'content': 'Exception!'}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
Since all exceptions are the same, and they all cause the function to return early, would it be suitable to wrap the entire function body in a big `try` `except` block instead of having 3 separate blocks? Not applicable if you're going to be specializing each one obviously.